src/gt4py/next/program_processors/runners/dace_fieldview/gtir_fieldview_builder.py

```diff
@@ -92,11 +97,20 @@ def visit_SymRef(self, node: itir.SymRef) -> None:
         self._ctx.add_input_node(dname)
 
     def write_to(self, node: itir.Expr) -> None:
+
+        # If I understand the code correctly, this function is to write `lhs := rhs` the value that is
```

Comment:
Right. This is a very special case. I was thinking that the target expression could contain make_tuple, as it was in ITIR, but probably this is not true in fieldview. I will check this with Hannes.